releasenotes/notes/process-agent-clean-pid-file-d962518272908132.yaml

```diff
@@ -0,0 +1,3 @@
+fixes:
+  - |
+    Clean up pid file whenever os.Exit() is called.
```

Comment:
If you look at the release notes, they don't seem to be separated by agent type. I think we can have one release note that says "process-agent and system-probe now clean up their PID files when exiting"

Reply:
I've created separated files because I've seen separated releases for `apm` and `logs`. But I'll merge these two into a single one with the note that you've suggested! I'll use the name `process-clean-pid-file`to make it easier to find it later.